nidaqmx/utils.py

```diff
@@ -13,7 +15,16 @@
     "the colon. Colons are not allowed within the names of the individual "
     "objects.")
 
+def select_interpreter(grpc_options = None, interpreter = None):
```

Comment:
This is a public submodule containing public utils. Either add a leading underscore to select_interpreter to denote that it is private, or move it to a private module.